Patient-MSSC/src/main/java/com/clinics/patient/service/VisitServiceImpl.java

```diff
@@ -27,31 +33,73 @@ public VisitServiceImpl(PatientClient patientClient, VisitRepository visitReposi
     }
 
     @Override
-    public Visit findByUuid(UUID uuid) {
-        return visitRepository.findByUuid(uuid);
+    @Transactional
```

Comment:
Why do you use this annotation over the method and not the class?

Reply:
Yes, why do we need to use it over the whole class ?

Reply:
1) on the class you will have 1 annotation, not x
2) you make annotation 1 time and you can forget about it
3) but ... this annotation is tricky, if there will be problems with eg. save we will have to move back here, in doctor as well

Reply:
But not every method needs to be transactional